### ISSUE TITLE
[3.3.1q] Video is intermittently recorded even though recordVideo capability is false

### ISSUE THREAD

**Issue by joostvanwollingen** [zalando/zalenium#215]
When requesting multiple sessions in parallel with recordVideo=false, some sessions will have videos recorded regardless. Out of 10 sessions with recordVideo=false, 3 videos where recorded anyway. Perhaps this issue is related to https://github.com/zalando/zalenium/issues/214 ?

Kubernetes config and Java source I used to reproduce: https://github.com/joostvanwollingen/zalenium-kubernetes/tree/159229491f26d48a7c7d707af2e56144ffe7620a

See below for the full log.

Some snippets:

Creating the webdriver with recordVideo capability set to false

    @Before
    public void setUp() throws MalformedURLException {
        DesiredCapabilities capability = DesiredCapabilities.chrome();
        capability.setCapability("recordVideo", false);

        driver = new RemoteWebDriver(getGridUrlFromEnvironment(), capability);
        driver.get("http://www.google.com");
    }

All sessions are requested with recordVideo false, as can be seen in the logs
   
      Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]

Recording still happens:
    13:46:21.860 INFO - zalenium-40001-f3q8c [bash, -c, start-video]
    ....
    13:46:36.988 INFO - zalenium-40001-f3q8c video-rec: started
     Waiting for ffmpeg video recording to start...
    ...Videos at /home/seluser/videos/vid_chrome_40001.mp4* started to be recorded! (wait-video-rec.sh)

Full log
```
Kubernetes service account found.
-- LOG 13:40:55:223835549 Running additional checks...
Copying files for Dashboard...
cp: cannot create regular file '/home/seluser/videos/index.html': No such file or directory
Starting Nginx reverse proxy...
Starting Selenium Hub...
13:40:56.004 INFO - Selenium build info: version: '3.3.1', revision: '5234b32'
13:40:56.006 INFO - Launching Selenium Grid hub
13:40:57.201 INFO - binding de.zalando.ep.zalenium.servlet.LivePreviewServlet to /grid/admin/LivePreviewServlet/*
13:40:57.203 INFO - binding de.zalando.ep.zalenium.servlet.ZaleniumConsoleServlet to /grid/admin/ZaleniumConsoleServlet/*
13:40:57.207 INFO - binding de.zalando.ep.zalenium.servlet.ZaleniumResourceServlet to /grid/admin/ZaleniumResourceServlet/*
13:40:57.210 INFO - binding de.zalando.ep.zalenium.dashboard.DashboardCleanupServlet to /grid/admin/DashboardCleanupServlet/*
13:40:57.431 INFO - Will listen on 4445
13:40:57.962 INFO - Nodes should register to http://10.240.0.174:4445/grid/register/
13:40:57.972 INFO - Selenium Grid hub is up and running
...........Selenium Hub started!
Starting DockerSeleniumStarter node...
13:40:59.083 INFO - Selenium build info: version: '3.3.1', revision: '5234b32'
13:40:59.085 INFO - Launching a Selenium Grid node
2017-08-24 13:41:00.770:INFO::main: Logging initialized @2380ms to org.seleniumhq.jetty9.util.log.StdErrLog
13:41:00.922 INFO - Driver provider org.openqa.selenium.ie.InternetExplorerDriver registration is skipped:
 registration capabilities Capabilities [{ensureCleanSession=true, browserName=internet explorer, version=, platform=WINDOWS}] does not match the current platform LINUX
13:41:00.923 INFO - Driver provider org.openqa.selenium.edge.EdgeDriver registration is skipped:
 registration capabilities Capabilities [{browserName=MicrosoftEdge, version=, platform=WINDOWS}] does not match the current platform LINUX
13:41:00.923 INFO - Driver class not found: com.opera.core.systems.OperaDriver
13:41:00.924 INFO - Driver provider com.opera.core.systems.OperaDriver registration is skipped:
Unable to create new instances on this machine.
13:41:00.924 INFO - Driver class not found: com.opera.core.systems.OperaDriver
13:41:00.925 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
13:41:00.927 INFO - Driver provider org.openqa.selenium.safari.SafariDriver registration is skipped:
 registration capabilities Capabilities [{browserName=safari, version=, platform=MAC}] does not match the current platform LINUX
2017-08-24 13:41:01.022:INFO:osjs.Server:main: jetty-9.2.20.v20161216
2017-08-24 13:41:01.106:INFO:osjsh.ContextHandler:main: Started o.s.j.s.ServletContextHandler@4b5d6a01{/,null,AVAILABLE}
2017-08-24 13:41:01.160:INFO:osjs.AbstractConnector:main: Started ServerConnector@d16762e{HTTP/1.1,[http/1.1]}{0.0.0.0:30000}
2017-08-24 13:41:01.161:INFO:osjs.Server:main: Started @2770ms
13:41:01.161 INFO - Selenium Grid node is up and ready to register to the hub
13:41:01.214 INFO - Registering the node to the hub: http://localhost:4444/grid/register
13:41:02.157 INFO - Initialising Kubernetes support
13:41:02.320 INFO - SessionCleaner initialized with insideBrowserTimeout 0 and clientGoneTimeout 1800000 polling every 180000
13:41:04.290 INFO - Kubernetes support initialised.
	Pod name: zalenium-1613017923-629jl
	app label: zalenium
	zalenium service name: zalenium
	Kubernetes flavour: Vanilla Kubernetes
	Selenium Pod Resource Limits: {}
	Selenium Pod Resource Requests: {}
13:41:04.307 INFO - [DS] Capabilities grabbed from the docker-selenium image
13:41:04.323 INFO - The node is registered to the hub and ready to use
13:41:04.324 INFO - Registered a node http://10.240.0.174:30000
13:41:04.326 INFO - About to clean up any left over selenium pods created by zalenium
13:41:04.478 INFO - [DS]  Setting up 0 nodes...
13:41:04.481 INFO - 0 containers were created, it will take a bit more until all get registered.
...................DockerSeleniumStarter node started!
Sauce Labs not enabled...
Browser Stack not enabled...
TestingBot not enabled...
Zalenium is now ready!
*************************************** Data Processing Agreement ***************************************
By using this software you agree that the following non-PII (non personally identifiable information)
data will be collected, processed and used by Zalando SE for the purpose of improving our test
infrastructure tools. Anonymisation with respect of the IP address means that only the first two octets
of the IP address are collected.

See the complete license at https://github.com/zalando/zalenium/blob/master/LICENSE.md
*************************************** Data Processing Agreement ***************************************
13:45:17.885 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:45:17.893 INFO - [DS] Starting new node for {tz=Europe/Berlin, recordVideo=false, browserName=chrome, screenResolution=1900x1880, version=, platform=ANY}.
13:45:32.325 INFO - Registered a node http://10.240.0.175:40000
13:45:32.381 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:45:45.243 INFO - http://10.240.0.175:40000 start-video: Video recording is disabled
13:45:48.199 INFO - http://10.240.0.175:40000 AFTER_SESSION command received. Node should shutdown soon...
13:45:48.203 INFO - http://10.240.0.175:40000 stop-video: Video recording is disabled
13:45:48.217 INFO - zalenium-40000-kzwzn [bash, -c, transfer-logs.sh]
13:45:48.333 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:45:48.342 INFO - [DS] Starting new node for {tz=Europe/Berlin, recordVideo=false, browserName=chrome, screenResolution=1900x1880, version=, platform=ANY}.
13:45:48.854 INFO - zalenium-40000-kzwzn 
13:45:50.625 INFO - http://10.240.0.175:40000 Logs copied to: /home/seluser/videos/logs/zalenium_68997bc7-b053-494b-ae56-fded6d24960d_chrome_LINUX_20170824134532
13:45:50.962 INFO - http://10.240.0.175:40000 Marking the node as down because it was stopped after 1 tests.
13:45:50.971 INFO - http://10.240.0.175:40000 Marking the node as down because it was stopped after 1 tests.
13:45:50.981 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.175:40000
13:45:56.705 INFO - Registered a node http://10.240.0.176:40001
13:45:56.743 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:46:05.934 INFO - Registered a node http://10.240.2.47:40007
13:46:20.969 INFO - Registered a node http://10.240.0.177:40002
13:46:21.860 INFO - zalenium-40001-f3q8c [bash, -c, start-video]
13:46:34.297 INFO - Registered a node http://10.240.0.178:40003
13:46:36.988 INFO - zalenium-40001-f3q8c video-rec: started
Waiting for ffmpeg video recording to start...
...Videos at /home/seluser/videos/vid_chrome_40001.mp4* started to be recorded! (wait-video-rec.sh)

13:46:38.967 INFO - http://10.240.0.176:40001 AFTER_SESSION command received. Node should shutdown soon...
13:46:38.973 INFO - zalenium-40001-f3q8c [bash, -c, stop-video]
13:46:39.011 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:46:39.043 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:46:40.252 INFO - http://10.240.2.47:40007 start-video: Video recording is disabled
13:46:41.311 INFO - http://10.240.2.47:40007 AFTER_SESSION command received. Node should shutdown soon...
13:46:41.312 INFO - http://10.240.2.47:40007 stop-video: Video recording is disabled
13:46:41.378 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:46:41.411 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:46:41.423 INFO - zalenium-40007-99scr [bash, -c, transfer-logs.sh]
13:46:41.988 INFO - zalenium-40007-99scr 
13:46:43.855 INFO - http://10.240.2.47:40007 Logs copied to: /home/seluser/videos/logs/zalenium_662c34f2-1e29-4ef9-8705-0234116d30be_chrome_LINUX_20170824134639
13:46:43.957 INFO - http://10.240.2.47:40007 Marking the node as down because it was stopped after 1 tests.
13:46:43.967 INFO - http://10.240.2.47:40007 Marking the node as down because it was stopped after 1 tests.
13:46:43.968 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.2.47:40007
13:46:49.504 INFO - http://10.240.0.177:40002 start-video: Video recording is disabled
13:46:53.596 INFO - http://10.240.0.177:40002 AFTER_SESSION command received. Node should shutdown soon...
13:46:53.596 INFO - http://10.240.0.177:40002 stop-video: Video recording is disabled
13:46:53.597 INFO - zalenium-40002-lq9g2 [bash, -c, transfer-logs.sh]
13:46:53.716 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:46:53.761 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:46:54.107 INFO - zalenium-40002-lq9g2 
13:46:55.668 INFO - http://10.240.0.177:40002 Logs copied to: /home/seluser/videos/logs/zalenium_ea75a684-0e64-4fbd-b887-44955778b000_chrome_LINUX_20170824134641
13:46:55.880 INFO - http://10.240.0.177:40002 Marking the node as down because it was stopped after 1 tests.
13:46:55.888 INFO - http://10.240.0.177:40002 Marking the node as down because it was stopped after 1 tests.
13:46:55.889 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.177:40002
13:46:56.333 INFO - Registered a node http://10.240.0.179:40004
13:47:00.104 INFO - Registered a node http://10.240.0.180:40005
13:47:03.095 INFO - zalenium-40003-9vgw9 [bash, -c, start-video]
13:47:04.979 INFO - Registered a node http://10.240.0.181:40006
13:47:13.623 INFO - zalenium-40003-9vgw9 video-rec: started
Waiting for ffmpeg video recording to start...
..Videos at /home/seluser/videos/vid_chrome_40003.mp4* started to be recorded! (wait-video-rec.sh)

13:47:14.248 INFO - http://10.240.0.178:40003 AFTER_SESSION command received. Node should shutdown soon...
13:47:14.252 INFO - zalenium-40003-9vgw9 [bash, -c, stop-video]
13:47:14.435 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:47:14.447 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:47:16.914 INFO - zalenium-40001-f3q8c video-rec: stopped
Waiting for video to stop recording...
video-rec                        STOPPED   Aug 24 01:47 PM
Done waiting for video recording to stop.
Video recording stopped
--LOG 13:47:16:879117846 -- DEBUG: video-rec-stdout.log ----
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Press [q] to stop, [?] for help
frame=    2 fps=0.0 q=0.0 size=      72kB time=00:00:00.10 bitrate=5891.2kbits/s    
frame=    3 fps=1.3 q=0.0 size=      89kB time=00:00:01.30 bitrate= 561.6kbits/s    
frame=    4 fps=1.3 q=0.0 size=     133kB time=00:00:02.80 bitrate= 389.6kbits/s    
frame=    5 fps=1.3 q=0.0 size=     149kB time=00:00:03.50 bitrate= 349.8kbits/s    
frame=    6 fps=1.3 q=0.0 size=     166kB time=00:00:04.30 bitrate= 316.1kbits/s    
frame=    7 fps=1.3 q=0.0 size=     183kB time=00:00:05.00 bitrate= 299.1kbits/s    
frame=    8 fps=1.3 q=0.0 size=     199kB time=00:00:06.00 bitrate= 271.8kbits/s    
--LOG 13:46:44:114800928 Trapped SIGTERM or SIGINT so shutting down ffmpeg gracefully...
--LOG 13:46:44:213588753 Will kill VID_TOOL_PID=1423 ...
frame=    9 fps=1.3 q=0.0 size=     216kB time=00:00:06.90 bitrate= 255.9kbits/s    
frame=    9 fps=1.1 q=0.0 Lsize=     232kB time=00:00:07.70 bitrate= 246.7kbits/s    
video:231kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.349967%
[libx264 @ 0x1f670a0] frame I:1     Avg QP: 0.00  size: 72399
[libx264 @ 0x1f670a0] frame P:8     Avg QP: 0.00  size: 20459
[libx264 @ 0x1f670a0] mb I  I16..4: 100.0%  0.0%  0.0%
[libx264 @ 0x1f670a0] mb P  I16..4: 94.6%  0.0%  0.0%  P16..4:  0.0%  0.0%  0.0%  0.0%  0.0%    skip: 5.3%
[libx264 @ 0x1f670a0] coded y,u,v intra: 0.4% 0.2% 0.2% inter: 0.2% 0.1% 0.1%
[libx264 @ 0x1f670a0] i16 v,h,dc,p: 99%  1%  0%  0%
[libx264 @ 0x1f670a0] kb/s:224.83
Exiting normally, received signal 2.
--LOG 13:46:45:308011151 Tried to kill -SIGTERM VID_TOOL_PID=1423
--LOG 13:46:45:355032816 Waiting up to 6s for VID_TOOL_PID=1423 to end with SIGTERM...
--LOG 13:46:45:441687757 wait_pid successfully managed to SIGTERM:VID_TOOL_PID=1423 within less than 6s
--LOG 13:46:45:481788598 Will try to fix the videos...
--LOG 13:46:46:099680938 Fixing perms for /home/seluser/videos/vid_chrome_40001.mkv*
--LOG 13:46:46:213612010 Changing video encoding from mkv to mp4...
--LOG 13:47:07:007467691 Conversion from mkv to mp4 FAILED! in within the 20s
--LOG 13:47:07:089641272 Optimizing /home/seluser/videos/vid_chrome_40001.mp4 for HTTP streaming...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:07:277015361 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:08:405885435 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:09:557213263 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:10:666793825 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:11:772028084 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:12:865832035 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:13:927763178 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40001.mp4: Invalid IsoMedia File
--LOG 13:47:15:034848181 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
--LOG 13:47:15:181971894 Failed! to mp4box_retry.sh within 8s
--LOG 13:47:15:254914848 ffmpeg shutdown complete.
--LOG 13:47:16:898276306 -- DEBUG: video-rec-stderr.log ----
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 40.101 /  5. 40.101
  libavresample   2.  1.  0 /  2.  1.  0
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.101 /  1.  2.101
  libpostproc    53.  3.100 / 53.  3.100
Input #0, matroska,webm, from '/home/seluser/videos/vid_chrome_40001.mkv':
  Metadata:
    ENCODER         : Lavf56.40.101
  Duration: 00:00:07.70, start: 0.000000, bitrate: 246 kb/s
    Stream #0:0: Video: h264 (High 4:4:4 Predictive), yuv444p, 1900x1880, SAR 1:1 DAR 95:94, 10 fps, 10 tbr, 1k tbn, 20 tbc (default)
    Metadata:
      ENCODER         : Lavc56.60.100 libx264
      DURATION        : 00:00:07.700000000
No pixel format specified, yuv444p for H.264 encoding chosen.
Use -pix_fmt yuv420p for compatibility with outdated media players.
[libx264 @ 0x1377980] using SAR=1/1
[libx264 @ 0x1377980] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x1377980] profile High 4:4:4 Predictive, level 5.0, 4:4:4 8-bit
[libx264 @ 0x1377980] 264 - core 148 r2643 5c65704 - H.264/MPEG-4 AVC codec - Copyleft 2003-2015 - http://www.videolan.org/x264.html - options: cabac=0 ref=1 deblock=0:0:0 analyse=0:0 me=dia subme=0 psy=0 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=0 chroma_qp_offset=0 threads=1 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=0 weightp=0 keyint=250 keyint_min=10 scenecut=0 intra_refresh=0 rc=cqp mbtree=0 qp=0
Output #0, mp4, to '/home/seluser/videos/vid_chrome_40001.mp4':
  Metadata:
    encoder         : Lavf56.40.101
    Stream #0:0: Video: h264 (libx264) ([33][0][0][0] / 0x0021), yuv444p, 1900x1880 [SAR 1:1 DAR 95:94], q=-1--1, 10 fps, 10240 tbn, 10 tbc (default)
    Metadata:
      DURATION        : 00:00:07.700000000
      encoder         : Lavc56.60.100 libx264
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
Press [q] to stop, [?] for help
frame=   28 fps=5.7 q=0.0 size=     515kB time=00:00:02.70 bitrate=1561.8kbits/s dup=25 drop=0    
frame=   35 fps=3.1 q=0.0 size=     629kB time=00:00:03.40 bitrate=1514.5kbits/s dup=31 drop=0    
frame=   43 fps=3.0 q=0.0 size=     759kB time=00:00:04.20 bitrate=1480.2kbits/s dup=38 drop=0    
frame=   43 fps=2.3 q=0.0 Lsize=     776kB time=00:00:04.30 bitrate=1478.5kbits/s dup=38 drop=0    
video:775kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.125112%
[libx264 @ 0x1377980] frame I:1     Avg QP: 0.00  size: 72399
[libx264 @ 0x1377980] frame P:42    Avg QP: 0.00  size: 17161
[libx264 @ 0x1377980] mb I  I16..4: 100.0%  0.0%  0.0%
[libx264 @ 0x1377980] mb P  I16..4: 93.3%  0.0%  0.0%  P16..4:  0.0%  0.0%  0.0%  0.0%  0.0%    skip: 6.7%
[libx264 @ 0x1377980] coded y,u,v intra: 0.1% 0.0% 0.0% inter: 0.0% 0.0% 0.0%
[libx264 @ 0x1377980] i16 v,h,dc,p: 100%  0%  0%  0%
[libx264 @ 0x1377980] kb/s:1475.66
Exiting normally, received signal 15.

13:47:17.255 INFO - http://10.240.0.176:40001 Video file copied to: /home/seluser/videos/zalenium_595cb1d8-eb3c-4aab-a411-be8737671920_chrome_LINUX_20170824134717.mp4
13:47:18.260 INFO - zalenium-40001-f3q8c [bash, -c, transfer-logs.sh]
13:47:18.511 INFO - zalenium-40001-f3q8c 
13:47:19.018 INFO - http://10.240.0.179:40004 start-video: Video recording is disabled
13:47:19.821 INFO - http://10.240.0.176:40001 Logs copied to: /home/seluser/videos/logs/zalenium_595cb1d8-eb3c-4aab-a411-be8737671920_chrome_LINUX_20170824134717
13:47:19.897 INFO - http://10.240.0.176:40001 Marking the node as down because it was stopped after 1 tests.
13:47:19.901 INFO - http://10.240.0.176:40001 Marking the node as down because it was stopped after 1 tests.
13:47:19.903 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.176:40001
13:47:22.383 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:47:22.388 INFO - http://10.240.0.179:40004 AFTER_SESSION command received. Node should shutdown soon...
13:47:22.392 INFO - http://10.240.0.179:40004 stop-video: Video recording is disabled
13:47:22.393 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:47:22.400 INFO - zalenium-40004-zs7vd [bash, -c, transfer-logs.sh]
13:47:22.730 INFO - zalenium-40004-zs7vd 
13:47:24.142 INFO - http://10.240.0.179:40004 Logs copied to: /home/seluser/videos/logs/zalenium_771a21bd-026b-4638-b606-036282081fb6_chrome_LINUX_20170824134714
13:47:24.240 INFO - http://10.240.0.179:40004 Marking the node as down because it was stopped after 1 tests.
13:47:24.242 INFO - http://10.240.0.179:40004 Marking the node as down because it was stopped after 1 tests.
13:47:24.245 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.179:40004
13:47:28.900 INFO - http://10.240.0.180:40005 start-video: Video recording is disabled
13:47:32.813 INFO - http://10.240.0.180:40005 AFTER_SESSION command received. Node should shutdown soon...
13:47:32.820 INFO - http://10.240.0.180:40005 stop-video: Video recording is disabled
13:47:32.821 INFO - zalenium-40005-1p492 [bash, -c, transfer-logs.sh]
13:47:32.899 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:47:32.908 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:47:33.411 INFO - zalenium-40005-1p492 
13:47:34.962 INFO - http://10.240.0.180:40005 Logs copied to: /home/seluser/videos/logs/zalenium_7c62f58b-1ef5-4d72-ad01-cb146df22b27_chrome_LINUX_20170824134722
13:47:35.133 INFO - http://10.240.0.180:40005 Marking the node as down because it was stopped after 1 tests.
13:47:35.134 INFO - http://10.240.0.180:40005 Marking the node as down because it was stopped after 1 tests.
13:47:35.135 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.180:40005
13:47:39.509 INFO - zalenium-40003-9vgw9 video-rec: stopped
Waiting for video to stop recording...
video-rec                        STOPPED   Aug 24 01:47 PM
Done waiting for video recording to stop.
Video recording stopped
--LOG 13:47:39:421277306 -- DEBUG: video-rec-stdout.log ----
Input #0, x11grab, from ':79.0+0,0':
  Duration: N/A, start: 1503575233.259694, bitrate: N/A
    Stream #0:0: Video: rawvideo (BGR[0] / 0x524742), bgr0, 1900x1880, 10 fps, 10 tbr, 1000k tbn, 10 tbc
No pixel format specified, yuv444p for H.264 encoding chosen.
Use -pix_fmt yuv420p for compatibility with outdated media players.
[libx264 @ 0x20490a0] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x20490a0] profile High 4:4:4 Predictive, level 5.0, 4:4:4 8-bit
[libx264 @ 0x20490a0] 264 - core 148 r2643 5c65704 - H.264/MPEG-4 AVC codec - Copyleft 2003-2015 - http://www.videolan.org/x264.html - options: cabac=0 ref=1 deblock=0:0:0 analyse=0:0 me=dia subme=0 psy=0 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=0 chroma_qp_offset=0 threads=1 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=0 weightp=0 keyint=250 keyint_min=10 scenecut=0 intra_refresh=0 rc=cqp mbtree=0 qp=0
Output #0, matroska, to '/home/seluser/videos/vid_chrome_40003.mkv':
  Metadata:
    encoder         : Lavf56.40.101
    Stream #0:0: Video: h264 (libx264) (H264 / 0x34363248), yuv444p, 1900x1880, q=-1--1, 10 fps, 1k tbn, 10 tbc
    Metadata:
      encoder         : Lavc56.60.100 libx264
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Press [q] to stop, [?] for help
frame=    3 fps=0.0 q=0.0 size=      89kB time=00:00:00.50 bitrate=1450.5kbits/s    
frame=    4 fps=2.9 q=0.0 size=     133kB time=00:00:01.10 bitrate= 987.2kbits/s    
frame=    5 fps=2.3 q=0.0 size=     149kB time=00:00:01.70 bitrate= 717.2kbits/s    
frame=    7 fps=2.3 q=0.0 size=     182kB time=00:00:02.80 bitrate= 531.9kbits/s    
--LOG 13:47:17:509155890 Trapped SIGTERM or SIGINT so shutting down ffmpeg gracefully...
--LOG 13:47:17:551373494 Will kill VID_TOOL_PID=1717 ...
frame=    9 fps=2.3 q=0.0 size=     214kB time=00:00:03.80 bitrate= 462.0kbits/s    
frame=    9 fps=2.0 q=0.0 Lsize=     231kB time=00:00:04.30 bitrate= 439.9kbits/s    
video:230kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.344582%
[libx264 @ 0x20490a0] frame I:1     Avg QP: 0.00  size: 72445
[libx264 @ 0x20490a0] frame P:8     Avg QP: 0.00  size: 20335
[libx264 @ 0x20490a0] mb I  I16..4: 100.0%  0.0%  0.0%
[libx264 @ 0x20490a0] mb P  I16..4: 94.6%  0.0%  0.0%  P16..4:  0.0%  0.0%  0.0%  0.0%  0.0%    skip: 5.4%
[libx264 @ 0x20490a0] coded y,u,v intra: 0.4% 0.2% 0.2% inter: 0.1% 0.0% 0.0%
[libx264 @ 0x20490a0] i16 v,h,dc,p: 99%  1%  0%  0%
[libx264 @ 0x20490a0] kb/s:400.21
Exiting normally, received signal 2.
--LOG 13:47:18:593894028 Tried to kill -SIGTERM VID_TOOL_PID=1717
--LOG 13:47:18:630076127 Waiting up to 6s for VID_TOOL_PID=1717 to end with SIGTERM...
--LOG 13:47:18:728897683 wait_pid successfully managed to SIGTERM:VID_TOOL_PID=1717 within less than 6s
--LOG 13:47:18:771923995 Will try to fix the videos...
--LOG 13:47:19:333123176 Fixing perms for /home/seluser/videos/vid_chrome_40003.mkv*
--LOG 13:47:19:370815862 Changing video encoding from mkv to mp4...
--LOG 13:47:36:112600787 Conversion from mkv to mp4 succeeded!
--LOG 13:47:36:199120751 Cleaning up /home/seluser/videos/vid_chrome_40003.mkv ...
--LOG 13:47:36:265601855 Optimizing /home/seluser/videos/vid_chrome_40003.mp4 for HTTP streaming...
--LOG 13:47:36:544917445 Succeeded to mp4box_retry.sh within 8s
--LOG 13:47:36:630142144 ffmpeg shutdown complete.
--LOG 13:47:39:484057477 -- DEBUG: video-rec-stderr.log ----
  libavformat    56. 40.101 / 56. 40.101
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 40.101 /  5. 40.101
  libavresample   2.  1.  0 /  2.  1.  0
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.101 /  1.  2.101
  libpostproc    53.  3.100 / 53.  3.100
Input #0, matroska,webm, from '/home/seluser/videos/vid_chrome_40003.mkv':
  Metadata:
    ENCODER         : Lavf56.40.101
  Duration: 00:00:04.30, start: 0.000000, bitrate: 439 kb/s
    Stream #0:0: Video: h264 (High 4:4:4 Predictive), yuv444p, 1900x1880, SAR 1:1 DAR 95:94, 10 fps, 10 tbr, 1k tbn, 20 tbc (default)
    Metadata:
      ENCODER         : Lavc56.60.100 libx264
      DURATION        : 00:00:04.300000000
No pixel format specified, yuv444p for H.264 encoding chosen.
Use -pix_fmt yuv420p for compatibility with outdated media players.
[libx264 @ 0x1a6c480] using SAR=1/1
[libx264 @ 0x1a6c480] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x1a6c480] profile High 4:4:4 Predictive, level 5.0, 4:4:4 8-bit
[libx264 @ 0x1a6c480] 264 - core 148 r2643 5c65704 - H.264/MPEG-4 AVC codec - Copyleft 2003-2015 - http://www.videolan.org/x264.html - options: cabac=0 ref=1 deblock=0:0:0 analyse=0:0 me=dia subme=0 psy=0 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=0 chroma_qp_offset=0 threads=1 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=0 weightp=0 keyint=250 keyint_min=10 scenecut=0 intra_refresh=0 rc=cqp mbtree=0 qp=0
Output #0, mp4, to '/home/seluser/videos/vid_chrome_40003.mp4':
  Metadata:
    encoder         : Lavf56.40.101
    Stream #0:0: Video: h264 (libx264) ([33][0][0][0] / 0x0021), yuv444p, 1900x1880 [SAR 1:1 DAR 95:94], q=-1--1, 10 fps, 10240 tbn, 10 tbc (default)
    Metadata:
      DURATION        : 00:00:04.300000000
      encoder         : Lavc56.60.100 libx264
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
Press [q] to stop, [?] for help
frame=   11 fps=8.5 q=0.0 size=     243kB time=00:00:01.00 bitrate=1993.0kbits/s dup=8 drop=0    
frame=   17 fps=5.7 q=0.0 size=     341kB time=00:00:01.60 bitrate=1745.2kbits/s dup=13 drop=0    
frame=   24 fps=5.0 q=0.0 size=     455kB time=00:00:02.30 bitrate=1620.3kbits/s dup=19 drop=0    
frame=   28 fps=4.0 q=0.0 size=     520kB time=00:00:02.70 bitrate=1578.2kbits/s dup=22 drop=0    
frame=   34 fps=4.1 q=0.0 size=     618kB time=00:00:03.30 bitrate=1533.5kbits/s dup=27 drop=0    
frame=   38 fps=3.7 q=0.0 size=     683kB time=00:00:03.70 bitrate=1511.7kbits/s dup=30 drop=0    
frame=   43 fps=3.6 q=0.0 size=     764kB time=00:00:04.20 bitrate=1490.9kbits/s dup=34 drop=0    
frame=   43 fps=3.1 q=0.0 size=     764kB time=00:00:04.20 bitrate=1490.9kbits/s dup=34 drop=0    
frame=   46 fps=2.9 q=0.0 Lsize=     830kB time=00:00:04.60 bitrate=1478.8kbits/s dup=37 drop=0    
video:829kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.118335%
[libx264 @ 0x1a6c480] frame I:1     Avg QP: 0.00  size: 72445
[libx264 @ 0x1a6c480] frame P:45    Avg QP: 0.00  size: 17252
[libx264 @ 0x1a6c480] mb I  I16..4: 100.0%  0.0%  0.0%
[libx264 @ 0x1a6c480] mb P  I16..4: 94.1%  0.0%  0.0%  P16..4:  0.0%  0.0%  0.0%  0.0%  0.0%    skip: 5.9%
[libx264 @ 0x1a6c480] coded y,u,v intra: 0.1% 0.0% 0.0% inter: 0.0% 0.0% 0.0%
[libx264 @ 0x1a6c480] i16 v,h,dc,p: 100%  0%  0%  0%
[libx264 @ 0x1a6c480] kb/s:1476.11

13:47:40.176 INFO - http://10.240.0.178:40003 Video file copied to: /home/seluser/videos/zalenium_ec2b1ef3-9028-41cd-86a1-b7f19fabf111_chrome_LINUX_20170824134739.mp4
13:47:41.179 INFO - zalenium-40003-9vgw9 [bash, -c, transfer-logs.sh]
13:47:41.214 INFO - http://10.240.0.181:40006 start-video: Video recording is disabled
13:47:41.508 INFO - zalenium-40003-9vgw9 
13:47:42.864 INFO - http://10.240.0.178:40003 Logs copied to: /home/seluser/videos/logs/zalenium_ec2b1ef3-9028-41cd-86a1-b7f19fabf111_chrome_LINUX_20170824134739
13:47:43.008 INFO - http://10.240.0.178:40003 Marking the node as down because it was stopped after 1 tests.
13:47:43.013 INFO - http://10.240.0.178:40003 Marking the node as down because it was stopped after 1 tests.
13:47:43.013 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.178:40003
13:47:43.566 INFO - http://10.240.0.181:40006 AFTER_SESSION command received. Node should shutdown soon...
13:47:43.570 INFO - http://10.240.0.181:40006 stop-video: Video recording is disabled
13:47:43.570 INFO - zalenium-40006-ptw81 [bash, -c, transfer-logs.sh]
13:47:43.653 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:47:43.656 INFO - [DS] Starting new node for {tz=Europe/Berlin, recordVideo=false, browserName=chrome, screenResolution=1900x1880, version=, platform=ANY}.
13:47:43.894 INFO - zalenium-40006-ptw81 
13:47:45.174 INFO - http://10.240.0.181:40006 Logs copied to: /home/seluser/videos/logs/zalenium_62ff8670-7918-4e8a-97c4-16bde66cb7fd_chrome_LINUX_20170824134732
13:47:45.262 INFO - http://10.240.0.181:40006 Marking the node as down because it was stopped after 1 tests.
13:47:45.263 INFO - http://10.240.0.181:40006 Marking the node as down because it was stopped after 1 tests.
13:47:45.264 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.0.181:40006
13:48:19.233 INFO - Registered a node http://10.240.1.44:40014
13:48:19.305 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:48:23.157 INFO - http://10.240.1.44:40014 start-video: Video recording is disabled
13:48:25.259 INFO - http://10.240.1.44:40014 AFTER_SESSION command received. Node should shutdown soon...
13:48:25.311 INFO - Got a request to create a new session: Capabilities [{recordVideo=false, browserName=chrome, version=, platform=ANY}]
13:48:25.325 INFO - [DS] Starting new node for {tz=Europe/Berlin, recordVideo=false, browserName=chrome, screenResolution=1900x1880, version=, platform=ANY}.
13:48:25.385 INFO - http://10.240.1.44:40014 stop-video: Video recording is disabled
13:48:25.389 INFO - zalenium-40014-7rn0n [bash, -c, transfer-logs.sh]
13:48:25.868 INFO - zalenium-40014-7rn0n 
13:48:27.484 INFO - http://10.240.1.44:40014 Logs copied to: /home/seluser/videos/logs/zalenium_479c0970-22f1-4aac-b380-b2eaa79c28bf_chrome_LINUX_20170824134819
13:48:27.598 INFO - http://10.240.1.44:40014 Marking the node as down because it was stopped after 1 tests.
13:48:27.604 INFO - http://10.240.1.44:40014 Marking the node as down because it was stopped after 1 tests.
13:48:27.605 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.1.44:40014
13:48:27.794 WARN - [DS] Max. number of docker-selenium containers has been reached, no more will be created until the number decreases below 10.
13:48:27.826 INFO - [DS] No container was created, will try again in a moment...
13:48:32.667 WARN - [DS] Max. number of docker-selenium containers has been reached, no more will be created until the number decreases below 10.
13:48:32.673 INFO - [DS] No container was created, will try again in a moment...
13:48:34.231 INFO - Registered a node http://10.240.2.48:40017
13:48:34.335 INFO - Trying to create a new session on test slot {chrome_binary=/usr/bin/google-chrome-stable, seleniumProtocol=WebDriver, acceptSslCerts=true, screen-resolution=1900x1880, tz=Europe/Berlin, browserName=chrome, maxInstances=1, screenResolution=1900x1880, resolution=1900x1880, version=60.0.3112.101, platform=LINUX}
13:48:34.476 WARN - [DS] Max. number of docker-selenium containers has been reached, no more will be created until the number decreases below 10.
13:48:34.482 INFO - [DS] No container was created, will try again in a moment...
13:48:34.916 INFO - Registered a node http://10.240.1.45:40016
13:48:37.152 INFO - zalenium-40017-n5r6r [bash, -c, start-video]
13:48:38.709 INFO - http://10.240.2.48:40017 AFTER_SESSION command received. Node should shutdown soon...
13:48:38.717 INFO - zalenium-40017-n5r6r [bash, -c, stop-video]
13:48:39.516 INFO - zalenium-40017-n5r6r video-rec: started
Waiting for ffmpeg video recording to start...
........Videos at /home/seluser/videos/vid_chrome_40017.mp4* started to be recorded! (wait-video-rec.sh)

13:48:49.260 INFO - zalenium-40017-n5r6r video-rec: stopped
Waiting for video to stop recording...
video-rec                        STOPPED   Aug 24 01:48 PM
Done waiting for video recording to stop.
Video recording stopped
--LOG 13:48:49:228598230 -- DEBUG: video-rec-stdout.log ----
Use -pix_fmt yuv420p for compatibility with outdated media players.
[libx264 @ 0x14f90c0] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x14f90c0] profile High 4:4:4 Predictive, level 5.0, 4:4:4 8-bit
[libx264 @ 0x14f90c0] 264 - core 148 r2643 5c65704 - H.264/MPEG-4 AVC codec - Copyleft 2003-2015 - http://www.videolan.org/x264.html - options: cabac=0 ref=1 deblock=0:0:0 analyse=0:0 me=dia subme=0 psy=0 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=0 chroma_qp_offset=0 threads=1 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=0 weightp=0 keyint=250 keyint_min=10 scenecut=0 intra_refresh=0 rc=cqp mbtree=0 qp=0
Output #0, matroska, to '/home/seluser/videos/vid_chrome_40017.mkv':
  Metadata:
    encoder         : Lavf56.40.101
    Stream #0:0: Video: h264 (libx264) (H264 / 0x34363248), yuv444p, 1900x1880, q=-1--1, 10 fps, 1k tbn, 10 tbc
    Metadata:
      encoder         : Lavc56.60.100 libx264
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Could not write header for output file #0 (incorrect codec parameters ?): Immediate exit requested
Exiting normally, received signal 2.
--LOG 13:48:40:209061882 Tried to kill -SIGTERM VID_TOOL_PID=874
--LOG 13:48:40:220019542 Waiting up to 6s for VID_TOOL_PID=874 to end with SIGTERM...
--LOG 13:48:40:238003531 wait_pid successfully managed to SIGTERM:VID_TOOL_PID=874 within less than 6s
--LOG 13:48:40:247705390 Will try to fix the videos...
--LOG 13:48:40:778070889 Fixing perms for /home/seluser/videos/vid_chrome_40017.mkv*
--LOG 13:48:40:801977539 Changing video encoding from mkv to mp4...
--LOG 13:48:40:902226257 Conversion from mkv to mp4 FAILED! in within the 20s
--LOG 13:48:40:920062881 Optimizing /home/seluser/videos/vid_chrome_40017.mp4 for HTTP streaming...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:40:950022897 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:41:979118508 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:43:001241751 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:44:029026838 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:45:055212225 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:46:083134178 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:47:106984060 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
Error opening file /home/seluser/videos/vid_chrome_40017.mp4: IsoMedia File is truncated
--LOG 13:48:48:138987811 MP4Box got errors meaning the mp4 video file is corrupted, trying again...
--LOG 13:48:48:932677187 Failed! to mp4box_retry.sh within 8s
--LOG 13:48:48:945007740 ffmpeg shutdown complete.
--LOG 13:48:49:232797081 -- DEBUG: video-rec-stderr.log ----
/usr/bin/start-video-rec.sh: line 57: kill: (874) - No such process
ffmpeg version 2.8.11-0ubuntu0.16.04.1 Copyright (c) 2000-2017 the FFmpeg developers
  built with gcc 5.4.0 (Ubuntu 5.4.0-6ubuntu1~16.04.4) 20160609
  configuration: --prefix=/usr --extra-version=0ubuntu0.16.04.1 --build-suffix=-ffmpeg --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --cc=cc --cxx=g++ --enable-gpl --enable-shared --disable-stripping --disable-decoder=libopenjpeg --disable-decoder=libschroedinger --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librtmp --enable-libschroedinger --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxvid --enable-libzvbi --enable-openal --enable-opengl --enable-x11grab --enable-libdc1394 --enable-libiec61883 --enable-libzmq --enable-frei0r --enable-libx264 --enable-libopencv
  libavutil      54. 31.100 / 54. 31.100
  libavcodec     56. 60.100 / 56. 60.100
  libavformat    56. 40.101 / 56. 40.101
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 40.101 /  5. 40.101
  libavresample   2.  1.  0 /  2.  1.  0
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.101 /  1.  2.101
  libpostproc    53.  3.100 / 53.  3.100
[matroska,webm @ 0x1194320] Format matroska,webm detected only with low score of 1, misdetection possible!
[matroska,webm @ 0x1194320] EBML header parsing failed
/home/seluser/videos/vid_chrome_40017.mkv: Invalid data found when processing input

13:48:49.591 INFO - http://10.240.2.48:40017 Video file copied to: /home/seluser/videos/zalenium_459074af-2ce1-420b-9580-728a76f418af_chrome_LINUX_20170824134849.mp4
13:48:50.617 INFO - zalenium-40017-n5r6r [bash, -c, transfer-logs.sh]
13:48:50.941 INFO - zalenium-40017-n5r6r 
13:48:52.567 INFO - http://10.240.2.48:40017 Logs copied to: /home/seluser/videos/logs/zalenium_459074af-2ce1-420b-9580-728a76f418af_chrome_LINUX_20170824134849
13:48:52.736 INFO - http://10.240.2.48:40017 Marking the node as down because it was stopped after 1 tests.
13:48:52.738 INFO - http://10.240.2.48:40017 Marking the node as down because it was stopped after 1 tests.
13:48:52.739 WARN - Cleaning up stale test sessions on the unregistered node http://10.240.2.48:40017
13:49:05.210 INFO - Registered a node http://10.240.0.183:40009
13:49:08.440 INFO - Registered a node http://10.240.0.184:40010
13:49:13.207 INFO - Registered a node http://10.240.0.185:40011
13:49:17.611 INFO - Registered a node http://10.240.0.186:40012
```

**Comment by diemol** [comment]
Seems like a duplicate of #214
I will keep this issue open until we verify it.

Thanks for the detailed report @joostvanwollingen

**Comment by diemol** [comment]
Hi @joostvanwollingen,

We just released a fix for this with https://github.com/zalando/zalenium/releases/tag/3.3.1r. I tested it with the docker version but I have not done so with k8s, I'll check that tomorrow, but perhaps you can also give a the new release a try in k8s? Thanks.

**Comment by joostvanwollingen** [comment]
3.3.1r fixed the issue for me, videos are no longer being recorded if recordVideo=false on the capabilities.

**Comment by diemol** [comment]
That's great @joostvanwollingen
Thanks for the quick feedback. 
I'll close this issue now.
